packages/expo-sms/src/SMS.ts

```diff
@@ -1,19 +1,41 @@
-import { UnavailabilityError } from '@unimodules/core';
+/* eslint-disable no-unused-expressions */
+import { UnavailabilityError, Platform } from '@unimodules/core';
 
 import ExpoSMS from './ExpoSMS';
-import { SMSResponse } from './SMS.types';
+import { SMSAttachment, SMSResponse, SMSOptions } from './SMS.types';
 
 export { SMSResponse };
 
+function processAttachments(
+  attachments: SMSAttachment | SMSAttachment[] | undefined
+): SMSAttachment[] | null {
+  if (!attachments) {
+    return null;
+  }
+  attachments = Array.isArray(attachments) ? attachments : [attachments];
+  if (Platform.OS === 'android' && attachments.length > 1) {
```

Comment:
If someone forgets, we don't want to warn the end-user
```suggestion
  if (__DEV__ && Platform.OS === 'android' && attachments.length > 1) {
```